test/notification.js

```diff
@@ -530,35 +532,34 @@ exports.testGlobalOptOut = function(test)
 exports.testMessageWithoutLocalization = function(test)
 {
   let notification = {message: "non-localized"};
-  let texts = Notification.getLocalizedTexts(notification, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
   test.equal(texts.message, "non-localized");
   test.done();
 };
 
 exports.testLanguageOnly = function(test)
 {
-  let notification = {message: {fr: "fr"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
-  texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr");
+  let notification = {message: {en: "en"}};
+  test.equal(Utils.appLocale, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
+  test.equal(texts.message, "en");
   test.done();
 };
 
 exports.testLanguageAndCountry = function(test)
 {
-  let notification = {message: {"fr": "fr", "fr-CA": "fr-CA"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr-CA");
-  texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
+  let notification = {message: {"en": "en", "en-US": "en-US"}};
```

Comment:
This test is not too useful any more, please could you change it back to how it was before? (Except you'll have to assign  `Utils.appLocale` as you do in the `textMissingTranslation` test.

Reply:
You mean exactly how it was? like this:

exports.testLanguageAndCountry = function(test)
{
  let notification = {message: {"fr": "fr", "fr-CA": "fr-CA"}};
  let texts = Notification.getLocalizedTexts(notification, "fr-CA");
  test.equal(texts.message, "fr-CA");
  texts = Notification.getLocalizedTexts(notification, "fr");
  test.equal(texts.message, "fr");
  test.done();
};

?

It could actually be useful because it falls to the first condition in the localize function (the test before - testLanguageOnly - falls to the second condition).

Reply:
I think there should be a test or two for the localize function alone now, because, as far as I understand, I can't change the Utils.appLocale parameter of this function from the test, unless I call it directly. I think that in testMissingTranslation the localize function is initiated with the real value of Utils.appLocale - "en-US", and not with the value that I set - "fr"